ethcore/light/src/cht.rs

```diff
@@ -66,11 +69,11 @@ impl<DB: HashDB> CHT<DB> {
 	/// Generate an inclusion proof for the entry at a specific block.
 	/// Nodes before level `from_level` will be omitted.
 	/// Returns an error on an incomplete trie, and `Ok(None)` on an unprovable request.
-	pub fn prove(&self, num: u64, from_level: u32) -> trie::Result<Option<Vec<Bytes>>> {
+	pub fn prove(&self, num: u64, from_level: u32) -> ethtrie::Result<Option<Vec<Bytes>>> {
 		if block_to_cht_number(num) != Some(self.number) { return Ok(None) }
 
 		let mut recorder = Recorder::with_depth(from_level);
-		let t = TrieDB::new(&self.db, &self.root)?;
+		let t = TrieDB::<_, RlpCodec>::new(&self.db, &self.root)?;
```

Comment:
there ought to be a type alias for `TrieDB` and `TrieDBMut` in `patricia-trie` so we don't need explicit type annotations

Reply:
> in patricia-trie

You mean in `patricia-trie-ethereum`?

Reply:
yeah